### PR TITLE
Add architecture

### PR DESCRIPTION
**Pull request by schmidtw**

### CODE REVIEW

File: content/docs/introduction/overview.md [hunk 1]
```diff
@@ -17,11 +17,42 @@ of connected clients and billions of messages delivered daily.
...
 
+Xmidt is designed to provide message delivery to **lots** of always on connected
+devices.  Xmidt makes the cloud easier for CPE devices to participate in at
+scale and makes the cloud interactions with the CPE devices more cloud friendly.
+
+If you have a large (or potentially large) number of clients that can exist
+behind firewalls, NATs or other infrastructure you need to operate, Xmidt can
+be a good match.  The CPE devices do not need to expose services like telnet,
+SSH, SNMP or others that can be compromised.  Instead, the Xmidt client (Parodus)
+reaches out to your cloud via a known URL and securely ensure it is conntected
```
Comment (kristinaspring) [inline content/docs/introduction/overview.md]: typo in connected

File: content/docs/introduction/overview.md [hunk 2]
```diff
@@ -17,11 +17,42 @@ of connected clients and billions of messages delivered daily.
...
+ * Scytale - api service front end
+ * Caduceus - event delivery servicing agent
+ * Issuer - (optional) JWT issuer for authentication and authorization
+ * [Consul](https://www.consul.io/) - coordination service by HashiCorp
+ * [Prometheus](https://www.prometheus.io/) - (optional) metrics gathering
+ * [Trickster](https://github.com/Comcast/trickster) - (optional) metrics optimization
 
 ### When does it fit?
 
+Xmidt is designed to provide message delivery to **lots** of always on connected
```
Comment (kristinaspring) [inline content/docs/introduction/overview.md]: Not sure if we need to specify that the devices are always on if they should be continuously connected?

File: content/docs/introduction/overview.md [hunk 3]
```diff
@@ -17,11 +17,42 @@ of connected clients and billions of messages delivered daily.
...
+scale and makes the cloud interactions with the CPE devices more cloud friendly.
+
+If you have a large (or potentially large) number of clients that can exist
+behind firewalls, NATs or other infrastructure you need to operate, Xmidt can
+be a good match.  The CPE devices do not need to expose services like telnet,
+SSH, SNMP or others that can be compromised.  Instead, the Xmidt client (Parodus)
+reaches out to your cloud via a known URL and securely ensure it is conntected
+only to your cluster.
+
+Xmidt will provide you a highly scaleable and cost effective way to device
```
Comment (kristinaspring) [inline content/docs/introduction/overview.md]: maybe "to manage devices"?  or rephrase the whole sentence: "Xmidt provides a highly scalable and cost effective solution to device management".

File: content/docs/introduction/overview.md (hunk 2 above)
Comment (schmidtw) [reply content/docs/introduction/overview.md]: I like that distinction better, as it's more clear.

File: content/docs/introduction/overview.md (hunk 3 above)
Comment (schmidtw) [reply content/docs/introduction/overview.md]: Much better sentence structure.

File: content/docs/introduction/overview.md [hunk 4]
```diff
@@ -17,11 +17,41 @@ of connected clients and billions of messages delivered daily.
...
 
+Xmidt is designed to provide message delivery to **lots** of continuously connected
+devices.  Xmidt makes the cloud easier for CPE devices to participate in at
+scale and makes the cloud interactions with the CPE devices more cloud friendly.
+
+If you have a large (or potentially large) number of clients that can exist
+behind firewalls, NATs or other infrastructure you need to operate, Xmidt can
+be a good match.  The CPE devices do not need to expose services like telnet,
+SSH, SNMP or others that can be compromised.  Instead, the Xmidt client (Parodus)
+reaches out to your cloud via a known URL and securely ensure it is connected
```
Comment (joe94) [inline content/docs/introduction/overview.md]: *ensures

**Pull request merged**